cypress/integration/pages/articles/tests.js

```diff
@@ -26,10 +26,56 @@ const serviceHasFigure = service =>
 const serviceHasCaption = service => service === 'news';
 // TODO: Remove after https://github.com/bbc/simorgh/issues/2962
 const serviceHasCorrectlyRenderedParagraphs = service => service !== 'sinhala';
+
 const serviceHasTimestamp = service => service === 'news';
 
-const tests = ({ service }) =>
-  describe(`Tests`, () => {
+// For testing important features that differ between services, e.g. Timestamps.
+// Inline conditional logic limiting tests to only those service/s which are different are encouraged.
```

Comment:
```suggestion
// We recommend using inline conditional logic to limit tests to services which differ
```

Reply:
I haven't suggested the change for all the other files. I assume there'll be a global search and replace.